src/core_plugins/tile_map/public/kibana_map.js

```diff
@@ -100,7 +100,9 @@ export class KibanaMap extends EventEmitter {
       minZoom: options.minZoom,
       maxZoom: options.maxZoom,
       center: options.center ? options.center : [0, 0],
-      zoom: options.zoom ? options.zoom : 2
+      zoom: options.zoom ? options.zoom : 2,
+      renderer: L.canvas(),
+      zoomAnimation: false // Desaturate map tiles causes animation rendering artifacts
```

Comment:
ok to switch this off for now. 

Client-side desaturation is a problem. we need a base-layer on the server-side that's already desaturated and work towards vector mapping on the client.